README.md

```diff
@@ -0,0 +1,24 @@
+# Palette Picker 
+
+### Overview:
+
+### Tech Stack:
+
+### Deployment Information: 
+
+### Setup:
+
+### Endpoints:
+
+|  **Purpose** 	| **Prefix For All URL's: `http://localhost:3000/`**  	| **Verb**  	| **Request Body**  	|   **Sample Success Response** |
```

Comment:
This looks good, I did make the changes on the PATCH endpoints so that they just return a single object rather than array containing that single object. It looks like those are already updated here to reflect that.

Reply:
Sounds good. I noticed a few typos so I i'm going to correct them and re push.